Makefile

```diff
@@ -543,6 +556,12 @@ upload-preloaded-images-tar: out/minikube # Upload the preloaded images for olde
 push-storage-provisioner-image: storage-provisioner-image ## Push storage-provisioner docker image using gcloud
 	gcloud docker -- push $(STORAGE_PROVISIONER_IMAGE)
 
+.PHONY: push-kic-base-image
+push-kic-base-image: kic-base-image ## Push kic-base docker image using gcloud
+	(docker pull $(KIC_BASE_IMAGE) && (echo "Image already exist"; exit 1) || echo "Image doesn't exist in registry")
+	$(call user_confirm, 'Are you sure you want to push $(KIC_BASE_IMAGE) ?')
+	gcloud docker -- push $(KIC_BASE_IMAGE)
```

Comment:
lets add docker hub and github packages pushes as well. also make sure to pull master to get the latest upstream

Reply:
Hi @medyagh 

I've few questions:
Assuming the user is already logged in - all clean
What if `IN_CI=false`? Should I ask user to provide password & username or should I fail?
What is `IN_CI=true`? Should I try to read username/password from some env variables or fail?